components/Snackbar/snackbar.css

```diff
@@ -0,0 +1,128 @@
+
+:root {
+  --primary-color: #ff7474;
+  --primary-text-color: #ff5454;
+  --background-color: #fde3e3;
+  --border-color: #d3d3d3;
+}
+
+.font {
+  font-family: "Roboto", sans-serif;
+  font-size: 1rem;
+}
+
+.snackbar_main_container {
+  margin: 1rem 0.5rem;
+}
+
+.btn_container{
+  border: 1px solid var(--border-color);
+  padding: 4rem 2rem;
+  border-radius: 1rem;
+
+}
+
+.snackbar_container {
+  background-color: var(--background-color);
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  width: 30rem;
+  border-bottom: 4px solid var(--primary-color);
+  position: fixed;
+  bottom: 0.5rem;
+  padding: 1rem 1.5rem;
+  margin: 1rem;
+  border-radius: 1rem;
+  visibility: hidden;
+}
+
+.snackbar_container a {
+  color: var(--primary-text-color);
+  text-decoration: none;
+  font-size: 1.2rem;
+  padding: 0.5rem 0.7rem;
+}
+
+.snackbar_container a:hover {
+  background-color: var(--primary-text-color);
+  color: white;
+}
+
+.baseline_snackbar,
+.stacked_snackbar {
+  left: 50%;
+  transform: translate(-50%, 0px);
+}
+
+.leading_snackbar{
+  left: 1%
+}
+
+.btn {
+  padding: 0.7rem;
+  background-color: white;
+  margin: 1rem;
+  border: none;
+  cursor: pointer;
+  background-color: var(--primary-color);
+  color: white;
+  border-radius: 0.5rem;
+}
+
+.btn:hover {
+  background-color: var(--primary-text-color);
+}
+
+.snackbar_button {
+  margin-left: 10%;
+  display: flex;
+  align-items: center;
+}
+
+.snackbar_button a {
+  font-size: 0.9rem;
+}
+
+.stacked_snackbar {
+  width: 35rem;
+}
+
+@media only screen and (max-width: 600px) {
+  .font {
+    font-size: 0.8rem;
+  }
+
+  .snackbar_container {
+    width: 60%;
+    padding: 0.4rem;
+    margin: 0.1rem;
+    border-radius: 10px;
+    font-size: 0.5rem;
+  }
+
+  .baseline_snackbar {
+    display: flex;
+    justify-content: center;
+  }
+
+  .btn {
+    margin: 0.5rem 0.2rem;
+  }
+
+  .snackbar_container a {
+    padding: 0.3rem 0.4rem;
+  }
+
+  .stacked_snackbar {
+    width: 16rem;
+  }
+
+  .snackbar_button a {
+    font-size: 0.4rem;
+  }
+
+  .snackbar_button a:nth-child(1) {
+    width: 3rem;
```

Comment:
line 97 has %. matter of preference choice. try to use consistent values

Reply:
okay.